src/Client.php

```diff
@@ -20,23 +26,24 @@
  */
 class Client{
 
-    private const WSDL_URL = "https://api.hostedshop.dk/service.wsdl";
+    private const WSDL_URL = 'https://api.hostedshop.dk/service.wsdl';
```

Comment:
This is a general trend - double quotes replaced by singles. Why is that? If it's a matter of preference I don't see how this benefits anyone.